etc/listproviders/event.upload.asset.options.properties

```diff
@@ -31,16 +31,16 @@ list.name=eventUploadAssetOptions
 
 # Asset upload options are for new and existing events.
 # EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.CLASS_HANDOUT_NOTES={"id": "attachment_class_handout_notes", "type": "attachment", "flavorType": "attachment", "flavorSubType": "notes", "displayOrder":1}
-# EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.CAPTIONS_DFXP={"id":"attachment_captions_dfxp", "type": "attachment", "flavorType": "captions", "flavorSubType": "dfxp", "displayOrder":2, "accept": ".ttml, .dfxp, .xml"}
-# EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.CAPTIONS_WEBVTT={"id":"attachment_captions_webvtt", "type": "attachment", "flavorType": "text", "flavorSubType": "vtt", "displayOrder":3, "accept": ".vtt"}
-# EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.PREVIEW_IMAGE={"id":"attachment_preview_image", "type":"attachment", "flavorType": "presenter","flavorSubType": "search+preview", "displayOrder":4, "accept": ".bmp,.gif,.jpeg,.jpg,.png,.tif,.tiff"}
+# EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.PREVIEW_IMAGE={"id":"attachment_preview_image", "type":"attachment", "flavorType": "presenter","flavorSubType": "search+preview", "displayOrder":2, "accept": ".bmp,.gif,.jpeg,.jpg,.png,.tif,.tiff"}
+# EVENTS.EVENTS.NEW.UPLOAD_ASSET.OPTION.SUBTITLES={"id": "track_subtitles_option", "type":"track", "flavorType": "captions","flavorSubType": "source", "tags": "generator:unknown", "displayOrder":3, "accept": ".vtt"}
 EVENTS.EVENTS.NEW.UPLOAD_ASSET.WORKFLOWDEFID=publish-uploaded-assets
 
 # The video source track upload options are only for new events.
+EVENTS.EVENTS.NEW.SOURCE.UPLOAD.NON_SEGMENTABLE={"id": "track_presenter","type":"track", "flavorType": "presenter","flavorSubType": "source", "multiple":false, "displayOrder":1, "accept": ".avi,.flv,.m4v,.mkv,.mov,.mp4,.mpeg,.mpg,.ogv,.webm,.wmv,.flac,.m4a,.mp3,.ogg,.wav,.wma,.f4v"}
+EVENTS.EVENTS.NEW.SOURCE.UPLOAD.SEGMENTABLE={"id": "track_presentation","type":"track", "flavorType": "presentation","flavorSubType": "source", "multiple":false, "displayOrder":2, "accept": ".avi,.flv,.m4v,.mkv,.mov,.mp4,.mpeg,.mpg,.ogv,.webm,.wmv,.flac,.m4a,.mp3,.ogg,.wav,.wma,.f4v"}
+EVENTS.EVENTS.NEW.SOURCE.UPLOAD.SUBTITLES={"id": "track_subtitles", "type":"track", "flavorType": "captions", "flavorSubType": "source", "tags": "generator:false", "displayOrder":3, "accept": ".vtt"}
```

Comment:
Suggestion: use `NEW.UPLOAD_ASSET` instead of `NEW.SOURCE` for subtitles. This would allow you to upload files afterward + during the new event dialog. Also stops new events that only contain subtitles.

Reply:
Wanted to reply to this earlier and then forgot. But now, finally, it is time.

I got two reasons for using `NEW.SOURCE` instead of `NEW.UPLOAD_ASSET`. The first is that the admin ui does not display assets of type `track` in the `NEW.UPLOAD_ASSET` tab of the new event dialog. Relevant code here: https://github.com/opencast/opencast/blob/a84dd5f0fac4f9883c9c621c7b82e07c11f96a51/modules/admin-ui-frontend/app/scripts/shared/services/wizards/new-event/uploadAssetNewEvent.js#L112. Maybe that is a design decision we want to rethink, maybe not, I did not want to open that can of worms.

The second reason is that grouping subtitle upload together with presenter and presentation is in my opinion less likely to annoy users who will not upload subtitles with every new event. Since I am proposing these settings as the new default, using `NEW.UPLOAD_ASSET` would require users to click through an additional tab everytime the want to create a new event, a tab which is useless to them if they do not wish to upload subtitles. I wanted to avoid this dent in the user experience (well, or they could just remove subtitles form this file, but UX is all about sensible defaults, right?).

As for new events that only contain subtitles, I am hopefully that that is a minor issue at best. Adding a validator that checks if the source uploads contain at least one media file would hopefully be fairly easy to achieve.